### ISSUE TITLE
NDS v5.0.0 fails to create deb package on Linux

### ISSUE THREAD

**Issue by martignoni** [nodogsplash/nodogsplash#577]
### Steps to reproduce

1. `git clone https://github.com/nodogsplash/nodogsplash.git`
2. `cd nodogsplash/`
3. `make deb`

### Expected behaviour

Deb package is built without error.

### Observed result

Build fails at step `dpkg-source -b .` with following error:
```
dpkg-source: error: can't build with source format '3.0 (native)': native package version may not have a revision
dpkg-buildpackage: error: dpkg-source -b . subprocess returned exit status 255
make: *** [Makefile:79: deb] Error 255
```
Hint to solution: in early stage of build, following line is output, hinting to problems into file `debian/changelog`:
```
mkdir -p dist/nodogsplash-3.3.2
```

**Comment by mwarning** [comment]
Thanks for helping out! :-)

I can confirm the problem. Could you use a more verbose commit message? Maybe "fix debian package build\nfixes #577".

**Comment by martignoni** [comment]
Is the commit change still needed (I'll do it with pleasure in this case), since you already merge the PR?
